test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoCreateForSuiteRunner.java

```diff
@@ -50,7 +51,7 @@
 	private String defaultNode;
 
 	//Use more initialBalance for this account as it is used as payer for the performance tests
-	private static OptionalLong initialBalance = OptionalLong.of(ONE_HBAR * 50000);
+	private static double initialBalance = 5.0 * (LoadTest.initialBalance.getAsLong());
```

Comment:
We should just use `long` and avoid the type cast later. All balances/fee are integers in tinybars, no floating-points